store/src/java/com/zimbra/cs/service/admin/GetDomainInfo.java

```diff
@@ -140,6 +140,10 @@ private void toXML(Element e, Entry entry, boolean applyConfig, boolean hasAuth)
                 for (String attr : bootstrapInfoAttrs) {
                     addAttrElementIfNotNull(entry, domain, attr);
                 }
+                if (entry instanceof Domain) {
+                    Domain d = (Domain)entry;
```

Comment:
Please rename variable d

Reply:
Thats fine but d as variable name will not look good in public repository, ideally it should be a proper name